src/Serialization/basic_types.jl

```diff
@@ -29,3 +43,13 @@ end
 function load_internal(s::DeserializerState, ::Type{String}, str::String)
     return str
 end
+
+################################################################################
+# Symbol
+function save_internal(s::SerializerState, sym::Symbol)
+    return string(sym)
+end
+
+function load_internal(s::SerializerState, ::Type{Symbol}, str::String)
```

Comment:
```suggestion
function load_internal(s::DeserializerState, ::Type{Symbol}, str::String)
```
Just a typo, should resolve the CI bugs.